v2/emailpassword/common-customizations/sessions/multiple-api-endpoints.mdx

```diff
@@ -52,6 +52,7 @@ SuperTokens.init({
         Session.init({
             // highlight-next-line
             cookieDomain: ".example.com",
+            olderCookieDomain: "" // Set to an empty string if your previous cookieDomain was unset. Otherwise, use your old cookieDomain value.
```

Comment:
Defaulting to undefined (or commented out or whatever) could be better. Also consider something like this (not the right wording and I'm not sure if it's better anyway):
```
// setting olderCookieDomain can be skipped if you will not have older session
// olderCookieDomain: "", // Set this to an empty string if the your previous cookieDomain was unset
// olderCookieDomain: "api.example.com" // Set this to the previous cookieDomain value if it was set
```

Reply:
Another option is to add setting the `olderCookieDomain` into a separate step as that gives you a bit more room for explanation and to explain all the implications